### ISSUE TITLE
Blocker issue and v6.0.0

### ISSUE THREAD

**Issue by simluk** [domaindrivendev/Swashbuckle.AspNetCore#1934]
Hello,

We are in need to get official release (patch) for following [bug](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/1836). It was said that v6.0.0 release will be after few weeks in october, while now it's almost 2021. It is blocking us from any updates of Swashbuckle.AspNetCore package. Please let me know when you plan to release it, via patch or major release.

Simonas

**Comment by ggleo08** [comment]
Get the same mistakes now , expect to fix this soon, aha.

     1.  more X-Forwarded-Host error "UriFormatException: Invalid URI: The hostname could not be parsed."
     2.  more x-forwarded-proto error "System.ArgumentException: Invalid URI: The URI scheme is not valid. (Parameter 'value')"

i add a middleware for avoid this problem。
```
    /// <summary>
    /// 自定义重置 X-Forwarded-Host 中间件
    /// TODO：fix Swashbuckle v5.6.3 bug，see https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/1836
    /// more X-Forwarded-Host error "UriFormatException: Invalid URI: The hostname could not be parsed."
    /// more x-forwarded-proto error "System.ArgumentException: Invalid URI: The URI scheme is not valid. (Parameter 'value')"
    /// </summary>
    public class XForwardedHostReplace
    {
        RequestDelegate _next;
        ILogger _logger;
        public XForwardedHostReplace(RequestDelegate next, ILogger<XForwardedHostReplace> logger)
        {
            _next = next;
            _logger = logger;
        }

        public async Task InvokeAsync(HttpContext context)
        {
            if (context.Request.Headers.TryGetValue("X-Forwarded-Host", out StringValues forwardedHost))
            {
                _logger.LogInformation($"X-Forwarded-Host：{forwardedHost} forwardedHost[0]：{ forwardedHost[0]}");
                var hostArray = forwardedHost[0].Split(",");
                if (hostArray.Length > 1)
                {
                    context.Request.Headers.Remove("X-Forwarded-Host");
                    context.Request.Headers.Add("X-Forwarded-Host", hostArray[0]);
                }
            }

            if (context.Request.Headers.TryGetValue("x-forwarded-proto", out StringValues forwardedProto))
            {
                _logger.LogInformation($"x-forwarded-proto：{forwardedHost} forwardedProto[0]：{ forwardedProto[0]}");
                var protoArray = forwardedProto[0].Split(",");
                if (protoArray.Length > 1)
                {
                    context.Request.Headers.Remove("x-forwarded-proto");
                    context.Request.Headers.Add("x-forwarded-proto", protoArray[0]);
                }
            }
            await _next(context);
        }
    }
```

**Comment by domaindrivendev** [comment]
`6.0.0` now published to Nuget
